### PR TITLE
User-guide: French translation

### PR DESCRIPTION
**Pull request by anonimal**
Referencing https://github.com/anonimal/kovri-site/pull/3.

### CODE REVIEW

File: kovri.i2p/_i18n/fr/user_guide.md [hunk 1]
```diff
@@ -1,81 +1,79 @@
...
+3. Rendez-vous sur http://check.kovri.i2p pour vérifier que cela fonctionne
 
 Notes:
 
-- **Just like with Tor, one doesn't need SSL to safely and securely use the network**
-- SSL site support and outproxy service is not currently implemented
-- If someone gives you a .i2p address that's not in your address book, use the `Jump` service at http://stats.i2p/i2p/lookup.html
-- Look through hosts.txt in your data directory to view a list of default sites you can easily visit
-- Overall, HTTP Proxy and address book implementation are in development and not yet feature-complete
+- **Exactement de la même façon que Tor, SSL n'est pas nécessaire pour utiliser sans risque et de façon sûre notre réseau**
```
Comment (MoroccanMalinois) [inline kovri.i2p/_i18n/fr/user_guide.md]: well, it's not really "our network" :), i would change "notre réseau" with "le réseau"

File: kovri.i2p/_i18n/fr/user_guide.md [hunk 2]
```diff
@@ -1,81 +1,79 @@
...
+- Le protocole SSL et le paramètre outproxy ne sont pas implantés pour le moment
+- Si quelqu'un vous donne une adresse .i2p qui n'est pas dans votre carnet d'adresses, utilisez le service `Jump` à l'adresse http://stats.i2p/i2p/lookup.html
+- Jetez un oeil au fichier hosts.txt dans votre répertoire de données pour voir une liste de sites par défaut que vous pouvez visiter. The docker image comes with the defaults of kovri,
+- Enfin, le Proxy HTTP et l'implémentation de carnets d'adresses sont en cours de développemment et ne sont pas complètement terminés.
```
Comment (MoroccanMalinois) [inline kovri.i2p/_i18n/fr/user_guide.md]: L'implémentation du Proxy HTTP et du carnet d'adresse sont en cours ....

File: kovri.i2p/_i18n/fr/user_guide.md [hunk 3]
```diff
@@ -1,81 +1,79 @@
...
 
 Notes:
 
-- **Just like with Tor, one doesn't need SSL to safely and securely use the network**
-- SSL site support and outproxy service is not currently implemented
-- If someone gives you a .i2p address that's not in your address book, use the `Jump` service at http://stats.i2p/i2p/lookup.html
-- Look through hosts.txt in your data directory to view a list of default sites you can easily visit
-- Overall, HTTP Proxy and address book implementation are in development and not yet feature-complete
+- **Exactement de la même façon que Tor, SSL n'est pas nécessaire pour utiliser sans risque et de façon sûre notre réseau**
+- Le protocole SSL et le paramètre outproxy ne sont pas implantés pour le moment
```
Comment (MoroccanMalinois) [inline kovri.i2p/_i18n/fr/user_guide.md]: > et le paramètre

et le service

> implantés

implémentés

File: kovri.i2p/_i18n/fr/user_guide.md [hunk 4]
```diff
@@ -1,81 +1,79 @@
...
+- Si quelqu'un vous donne une adresse .i2p qui n'est pas dans votre carnet d'adresses, utilisez le service `Jump` à l'adresse http://stats.i2p/i2p/lookup.html
+- Jetez un oeil au fichier hosts.txt dans votre répertoire de données pour voir une liste de sites par défaut que vous pouvez visiter. The docker image comes with the defaults of kovri,
+- Enfin, le Proxy HTTP et l'implémentation de carnets d'adresses sont en cours de développemment et ne sont pas complètement terminés.
 
-## Step 6. Host your own garlic-service (garlic-site/eepsite)
-- Read `tunnels.conf` to learn how to set a server tunnel to point to the service you are hosting
+## Etape 6. Organiser votre propre service garlic (garlic-site/eepsite)
```
Comment (MoroccanMalinois) [inline kovri.i2p/_i18n/fr/user_guide.md]: > Organiser votre

Hébergez votre

File: kovri.i2p/_i18n/fr/user_guide.md [hunk 5]
```diff
@@ -1,81 +1,79 @@
...
+- Lisez `tunnels.conf` pour apprendre comment définir un tunnel de serveur pour indiquer le service que vous hébergez.
```
Comment (MoroccanMalinois) [inline kovri.i2p/_i18n/fr/user_guide.md]: > comment définir un tunnel de serveur pour indiquer le service que vous hébergez

comment définir un serveur tunnel qui pointe vers le service que vous hébergez

File: kovri.i2p/_i18n/fr/user_guide.md [hunk 6]
```diff
@@ -1,81 +1,79 @@
...
 
-## Step 7. Enjoy!
-- Read more about Kovri in the [Moneropedia](https://getmonero.org/knowledge-base/moneropedia/kovri).
-- Open your feature requests or report bugs on our [issues tracker](https://github.com/monero-project/kovri/issues)
-- Learn more about the I2P network on the [java I2P website](https://geti2p.net/en/docs)
+## Etape 7. Eclatez-vous!enjoy
```
Comment (MoroccanMalinois) [inline kovri.i2p/_i18n/fr/user_guide.md]: > Eclatez-vous!enjoy

Amusez-vous!

File: kovri.i2p/_i18n/fr/user_guide.md [hunk 7]
```diff
@@ -1,81 +1,79 @@
...
-
-## Step 2. Configuring / Open Firewall
-
-The docker image comes with the defaults of kovri, but can be configured as explained in earlier sections.
+## Etape 1. Installer Docker
+Installer Docker est externe à ces instructions, veuillez vous réferer à la [documentation docker](https://docs.docker.com/engine/installation/)
 
-You should choose a random port and open that port (see earlier sections).
+## Etape 2. Configurer / Ouvrir Firewall
+Docker est fourni avec les valeurs par défaut de kovri, cependant il peut être configurer comme expliqué dans les sections précedentes.
```
Comment (MoroccanMalinois) [inline kovri.i2p/_i18n/fr/user_guide.md]: > Docker est fourni

L'image docker est fournie 

> cependant il peut

cependant elle peut

**Comment by anonimal** [comment]
Thank you very much @MoroccanMalinois.

@Dridou will you make these changes?

**Comment by Dridou** [comment]
Sorry I've been just seeing that I've done a lot of fails, I really tried to do it correlty though ! Thanks for having done changes.
I have a question ,why does it written "verified" next to your merge pull request with @MoroccanMalinois and it was not written with mine? Just asking for information.
Thx btw @MoroccanMalinois for corrections!

**Comment by anonimal** [comment]
>I have a question ,why does it written "verified" next to your merge pull request with @MoroccanMalinois and it was not written with mine?

We have GPG verified commits. See https://help.github.com/articles/signing-commits-using-gpg/.

**Pull request merged**